modules/client/test/integration/client/decoding.test.ts

```diff
@@ -490,5 +495,79 @@ describe("Client", () => {
         bytesToHex(decodedUpgradeToAndCallParams.data),
       );
     });
+    it("Should decode an update proxy implementation and call action", async () => {
```

Comment:
```suggestion
    it("Should decode an apply installation action", async () => {
```

Proxy upgrades are much simpler